tests/real/community.rs

```diff
@@ -62,6 +62,77 @@ fn roundtrip() {
     assert_eq!(job.args(), &[Value::from("z")]);
 }
 
+#[test]
+#[cfg(feature = "tls")]
+fn roundtrip_tls() {
+    use native_tls::{Certificate, TlsConnector};
+    use std::{env, fs::File, io::Read};
+
+    if env::var_os("FAKTORY_URL").is_none() || env::var_os("FAKTORY_URL_SECURE").is_none() {
+        return;
+    }
+    let local = "roundtrip_tls";
+
+    let (tx, rx) = sync::mpsc::channel();
+    let tx = sync::Arc::new(sync::Mutex::new(tx));
+    let mut c = ConsumerBuilder::default();
+    c.hostname("tester".to_string()).wid(local.to_string());
+    {
+        let tx = sync::Arc::clone(&tx);
+        c.register(local, move |j| -> io::Result<()> {
+            tx.lock().unwrap().send(j).unwrap();
+            Ok(())
+        });
+    }
+
+    let mut cert = String::new();
+    let cert_path = env::current_dir()
+        .unwrap()
+        .join("docker")
+        .join("certs")
+        .join("faktory.local.crt");
+    File::open(cert_path)
+        .and_then(|mut f| f.read_to_string(&mut cert))
+        .unwrap();
```

Comment:
```suggestion
    let cert = std::fs::read_to_string(cert_path).unwrap();
```

Reply:
Updated. I guess should have first applied the suggestions and then make restructures the `/tests` dir.